certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+    "PubCS.TitleOrder.Api.Models.PartyReference": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "partyIdReference": {
+          "description": "Gets or Sets PartyIdReference",
+          "type": "integer"
+        },
+        "nickName": {
+          "description": "Gets or Sets NickName",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "status": {
+          "type": "string"
+        },
+        "fileNumber": {
+          "description": "Gets or Sets FileNumber",
+          "type": "string"
+        },
+        "transactionId": {
+          "description": "Gets or Sets TransactionId",
+          "type": "integer"
+        },
+        "referenceKey": {
+          "description": "Gets or Sets ReferenceKey",
+          "type": "string"
+        },
+        "otherEnterpriseFileNumber": {
+          "description": "Gets or Sets OtherEnterpriseFileNumber",
+          "type": "string"
+        },
+        "association": {
+          "type": "string"
+        },
+        "service": {
+          "type": "string"
+        },
+        "type": {
+          "description": "Gets or Sets Type",
+          "type": "string"
+        },
+        "category": {
+          "description": "Gets or Sets Category",
+          "type": "string"
+        },
+        "template": {
+          "description": "Gets or Sets Template",
+          "type": "string"
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Date"
+          }
+        },
+        "titleCompany": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "region": {
+          "description": "Gets or Sets Region",
+          "type": "string"
+        },
+        "branch": {
+          "description": "Gets or Sets Branch",
+          "type": "string"
+        },
+        "agents": {
+          "description": "Gets or Sets Agents",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+          }
+        },
+        "underwriter": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "escrowBank": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "sourceOfBusiness": {
+          "description": "Gets or Sets SourceOfBusiness",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.ProductDetail": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.